jobs/mysql_backup_job.go

```diff
@@ -146,43 +173,42 @@ func (j *MySQLBackupJob) Run(ctx context.Context, send Send) (rerr error) {
 			errBackupBuffer.String(), outBuffer.String(), errCloudBuffer.String())
 	}
 
+	if err := xtrabackupCmd.Start(); err != nil {
+		cancel()
```

Comment:
Here I added `cancel` to prvent errors during copy-paste. So technically it's not necessary here, but it you will add some commands before this command it will work anyway.